ui/app/templates/components/okta-number-challenge.hbs

```diff
@@ -2,42 +2,36 @@
   Copyright (c) HashiCorp, Inc.
   SPDX-License-Identifier: BUSL-1.1
 ~}}
-
```

Comment:
### before 
<img width="696" alt="Screenshot 2024-06-11 at 3 59 48 PM" src="https://github.com/hashicorp/vault/assets/68122737/1740d80b-a82f-4f52-af80-87bdd02ac611">

Reply:
Curious what the before/after is when unauth listed auth mounts are present in tabs. Reading the code, it seems like before they were hidden on Okta verification and now they will be shown?

Reply:
Great question! As part of the setup, `okta` is configured to be a tab, and it's hidden here. The tabs are still rendered by `AuthForm` which is hidden completely when we're polling for the okta number response

https://github.com/hashicorp/vault/blob/2e3af257a9cdd40969c18ce7ae5fd2091a143c63/ui/app/templates/components/auth-form.hbs#L7-L41